packages/ui/src/components/Box/Box.tsx

```diff
@@ -14,13 +14,14 @@ export interface BoxOwnProps {
 	heading?: ReactNode
 	isActive?: boolean
 	intent?: Intent
+	padding?: Default | 'no-padding'
 }
 
 export interface BoxProps extends BoxOwnProps, Omit<NativeProps<HTMLDivElement>, 'children'> {}
 
 export const Box = memo(
 	forwardRef<HTMLDivElement, BoxProps>(
-		({ actions, className, distinction, children, gap, heading, intent, isActive, ...divProps }: BoxProps, ref) => {
+		({ actions, className, distinction, children, gap, heading, intent, isActive, padding = 'default', ...divProps }: BoxProps, ref) => {
```

Comment:
Is there a reason `padding = Default` not being used?

Reply:
`Default` is a type. See https://github.com/contember/admin/blob/pr/fix-table-popups/packages/ui/src/types/Default.ts#L1

Reply:
However, there needs to be no default value, so I'm removing it anyways. THanks for pointing out.

Reply:
> Default is a type

I see. My bad. I'm sorry.